.gitignore

```diff
@@ -5,6 +5,7 @@ compiled/
 dist/
 build/
 coverage/
+release/
```

Comment:
We need to have the release directory in the repository. Let's remove this line please. However, do not commit anything in the release directory. We generate that when we release a new version.